### PR TITLE
Applications page

### PR DESCRIPTION
**Pull request by tuler**
Page to list applications.
Still have to implement pagination.

### CODE REVIEW

**Comment by brunomenezes** [comment]
@tuler, 
I am getting an `Application error: a client-side exception has occurred (see the browser console for more information).` when accessing the `/applications`.  Also, it needs a rebase since it's set up on `pnpm`.

**Comment by nevendyulgerov** [comment]
Hey @tuler , @brunomenezes , I rebased the branch, fixed the client side error happening in `Address` component and implemented pagination.

File: apps/web/src/app/applications/page.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                                                    <TbInbox />
+                                                </ActionIcon>
+                                            </Link>
+                                        </Tooltip>
+                                    </Table.Td>
+                                </Table.Tr>
+                            ),
+                        )}
+                    </Table.Tbody>
+                    <Table.Tfoot></Table.Tfoot>
```
Comment (dandheedge) [inline apps/web/src/app/applications/page.tsx:141]: Can we remove this element? Since it's empty anyway

**Comment by tuler** [comment]
LGTM. Can we have another review?

**Review by dandheedge — APPROVED** [review]

**Pull request merged**